src/main/java/sirius/db/mixing/EntityRef.java

```diff
@@ -187,7 +187,7 @@ public boolean isValueLoaded() {
      * @return <tt>true</tt> if an entity is referenced (the stored id is not <tt>null</tt>). <tt>false</tt> otherwise
      */
     public boolean isFilled() {
-        return id != null;
+        return id != null && id != -1L;
```

Comment:
was passiert wenn die value des entityRefs ein noch nicht persistiertes Objekt ist? des würde hier ja false zurückliefern... richtig oder falsch?

Reply:
jo das passt: nicht persistiert is nicht existent